reconcile/utils/gql.py

```diff
@@ -173,10 +163,20 @@ def get_queried_schemas(self):
 
 def init(url, token=None, integration=None, validate_schemas=False):
     global _gqlapi
-    _gqlapi = GqlApi(url, token, integration, validate_schemas)
+    client = _init_gql_client(url, token)
+    _gqlapi = GqlApi(client, integration, validate_schemas)
     return _gqlapi
 
 
+def _init_gql_client(url, token) -> Client:
```

Comment:
For type hints, you want to add them for all parameters in addition to the return value.